tests/apichecks/testdata/exceptions/acronyms.txt

```diff
@@ -77,7 +77,8 @@
 [acronyms] crd=cloudfunctionsfunctions.cloudfunctions.cnrm.cloud.google.com version=v1beta1: field ".spec.sourceArchiveUrl" should be ".spec.sourceArchiveURL"
 [acronyms] crd=cloudfunctionsfunctions.cloudfunctions.cnrm.cloud.google.com version=v1beta1: field ".status.sourceRepository.deployedUrl" should be ".status.sourceRepository.deployedURL"
 [acronyms] crd=cloudfunctionsfunctions.cloudfunctions.cnrm.cloud.google.com version=v1beta1: field ".status.versionId" should be ".status.versionID"
-[acronyms] crd=cloudidsendpoints.cloudids.cnrm.cloud.google.com version=v1alpha1: field ".status.endpointIp" should be ".status.endpointIP"
+[acronyms] crd=cloudidsendpoints.cloudids.cnrm.cloud.google.com version=v1alpha1: field ".status.observedState.endpointIp" should be ".status.observedState.endpointIP"
+[acronyms] crd=cloudidsendpoints.cloudids.cnrm.cloud.google.com version=v1beta1: field ".status.observedState.endpointIp" should be ".status.observedState.endpointIP"
```

Comment:
I'm going to fast follow up this exception to rename the field `endpointIp` to `endpointIP`